src/components/Nav/index.js

```diff
@@ -393,24 +370,8 @@ const Nav = ({ handleThemeChange, isDarkTheme, path }) => {
             toggleTheme={handleThemeChange}
             linkSections={mobileLinkSections}
           />
-          <MobileIcons>
```

Comment:
I've moved all of this as it felt weird to have some mobile components here and some in `src/components/Nav/Mobile.js`. I think they should just live in the same place.

Reply:
Thanks for doing that =)